packages/apps/src/configs/storages/evm-storage.ts

```diff
@@ -0,0 +1,93 @@
+import { EvmChainStorage } from '@webb-dapp/apps/configs/storages/evm-chain-storage.interface';
+import { StorageHandler } from '@webb-dapp/utils';
+import { EVMStorage } from '@webb-dapp/react-environment/api-providers/web3';
+
+const rinkebyStore = new EvmChainStorage([
+  {
+    size: 0.1,
+    address: '0x876eCe69618e8E8dd743250B036785813824D2D7',
+    symbol: 'ETH',
+    syncedBlock: 120000, // should be hardcoded to deployed block number
+    leaves: [],
+  },
+]);
+
+const ethMainNet = new EvmChainStorage([
+  {
+    size: 0.1,
+    address: '0x876eCe69618e8E8dd743250B036785813824D2D7',
+    symbol: 'ETH',
+    syncedBlock: 1, 
+    leaves: [],
+  },
+]);
+
+const beresheet = new EvmChainStorage([
+  {
+    size: 10,
+    address: '0x5f771fc87F87DB48C9fB11aA228D833226580689',
+    symbol: 'tEDG',
+    syncedBlock: 1, 
+    leaves: [],
+  },
+  {
+    size: 100,
+    address: '0x2ee2e51cab1561E4482cacc8Be8b46CE61E46991',
+    symbol: 'tEDG',
+    syncedBlock: 1, 
+    leaves: [],
+  },
+  {
+    size: 1000,
+    address: '0x5696b4AfBc169454d7FA26e0a41828d445CFae20',
+    symbol: 'tEDG',
+    syncedBlock: 1, 
+    leaves: [],
+  },
+  {
+    size: 10000,
+    address: '0x626FEc5Ffa7Bf1EE8CEd7daBdE545630473E3ABb',
+    symbol: 'tEDG',
+    syncedBlock: 1, 
+    leaves: [],
+  },
+]);
+
+const defaultHandler: Omit<StorageHandler<EVMStorage>, 'inner'> = {
+  async fetch(key: string) {
+    const data = localStorage.getItem(key);
+    if (!data) {
+      return {
+        nativeAnchor: new EvmChainStorage([]),
+      };
+    }
+    const address = JSON.parse(data).contractsAddresses;
+    return {
+      nativeAnchor: new EvmChainStorage(address),
```

Comment:
Is this currently a nativeAnchor because the only mixers we currently have supported in the dApp are native? I suppose we'll want to plan accordingly for ERC20 supported mixers. Might be worth updating this now, thoughts?